src/js/components/stock-movement/outbound/StockMovementOutboundList.jsx

```diff
@@ -22,6 +23,10 @@ const StockMovementOutboundList = (props) => {
 
   useTranslation('stockMovement', 'StockMovementType', 'reactTable');
 
+  useEffect(() => {
+    triggerDeletionNotification(props.location?.search);
```

Comment:
I think you should also consider clearing that `deleted="true"` param from search query after extracting the value and displaying the notification.

Reply:
I didn't do that, because the param was deleted on list pages somehow, but to make this function more reusable, I will add this.

Reply:
it was cleared probably while fetching the list data, because there we also manipulate the query params, so maybe it was cleared then